docs/userman/gui/extension/extension_data.md

```diff
@@ -1,10 +1,211 @@
 # Using tabular data
-  
-!!! warning "Work in Progress"
-    This section still requires significant work, which is in progress.
-    At this time, Taipy GUI provides a custom element library example
-    with lengthy explanations on how to build it.<br/>
-    Please look into the `doc/extension` directory where Taipy GUI is
-    installed for more information.<br/>
-    You can also look at this example directly on
-    [GitHub](https://github.com/Avaiga/taipy/tree/[BRANCH]/doc/gui/extension).
+
```

Comment:
So this has the exact same title as in 'tabular_data_props.md'... why is that?

ooooh it seems that's a copy of the file... probably needs to go